### PR TITLE
After week 1 home screen

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Completed state changes for home screen after week 1.  

Re-organized the activities so that daily and sleep are always in the top row, and cognitive and physical challenges are now in the bottom row.

After week 1 the following UI changes happen...
1) The "today's activities expire in..." label now starts with "Daily activities expire in..."2) There is a new weekly expires in label that shows up above the cognitive and physical challenges.  On first 6 days of the week it says "Weekly activities expire in X days".  On the last day of the week it says "Weekly activities expire in 05:12:00" which is an hour, min, sec countdown like the daily expiration label.3) Instead of Day 1 of 7, that label should now show week number, like "Week 2"4) Physical and cognitive challenge actives should stay as marked "Done" with the checkmark until the next week.

See [Jira Issue](https://sagebionetworks.jira.com/browse/PKU-134) for a video of it working

### CODE REVIEW

File: BiomarinPKU/BiomarinPKU-Study/ActivityViewController.swift [hunk 1]
```diff
@@ -279,11 +288,66 @@ class ActivityViewController: UIViewController, RSDTaskViewControllerDelegate {
...
+    func expiresLabelText(for day: Int, expiresTimeStr: String) -> String? {
+        if day <= 7 {
+            return Localization.localizedStringWithFormatKey("WEEK_1_EXPIRES_FORMAT_%@", expiresTimeStr)
+        } else {
+            return Localization.localizedStringWithFormatKey("AFTER_WEEK_1_EXPIRES_FORMAT_%@", expiresTimeStr)
+        }
+    }
+    
+    func expiresWeeklyLabelText(for day: Int, week: Int, expiresTimeStr: String) -> String? {
+        if day <= 7 {
```
Comment (DanWebster) [inline BiomarinPKU/BiomarinPKU-Study/ActivityViewController.swift]: @dephillipsmichael brilliant

File: BiomarinPKU/BiomarinPKU/PKU.strings [hunk 2]
```diff
@@ -61,11 +61,16 @@
...
 "WEEK_1_TITLE" = "Let’s begin your journey!";
 "WEEK_1_TEXT" = "By completing your first 4 activities in a day for one week, you will then be able to unlock your entire journey.";
 "WEEK_1_EXPIRES_FORMAT_%@" = "Today’s activities expire in %@";
+"AFTER_WEEK_1_EXPIRES_FORMAT_%@" = "Daily activities expire in %@";
 "WEEK_1_ACTIVITY_SLEEP" = "Sleep Check-In";
+"AFTER_WEEK_1_EXPIRES_WEEKLY_FORMAT_DAYS_%@" = "Weekly activities expire in %@ days";
+"AFTER_WEEK_1_EXPIRES_WEEKLY_FORMAT_HOURS_%@" = "Weekly activities expire in %@";
 
 "ACTIVITY_PHYSICAL" = "Physical Challenge";
 "ACTIVITY_COGNITION" = "Cognition";
```
Comment (DanWebster) [inline BiomarinPKU/BiomarinPKU/PKU.strings]: Cognition -> Cognitive Challenge
To match the 'Physical Challenge' label

**Comment by dephillipsmichael** [comment]
@brian-bot made all the changes request and the ones mentioned by Woody in the meeting

**Comment by dephillipsmichael** [comment]
This also completes the new story https://sagebionetworks.jira.com/browse/PKU-135

**Comment by DanWebster** [comment]
> @dephillipsmichael - think you meant to tag @DanWebster and not me ;)

While I am flattered to be confused with @brian-bot, I would think that the converse of Brian getting confused by me would be an insult for the ages ;)

**Comment by dephillipsmichael** [comment]
> > @dephillipsmichael - think you meant to tag @DanWebster and not me ;)
> 
> While I am flattered to be confused with @brian-bot, I would think that the converse of Brian being mistaken as me would be an insult for the ages ;)

Haha!  My fault, too many simultaneous projects

File: BiomarinPKU/BiomarinPKU/PKU.strings [hunk 3]
```diff
@@ -61,14 +61,27 @@
...
 "WEEK_1_EXPIRES_FORMAT_%@" = "Today’s activities expire in %@";
+"WEEK_1_RENEW_FORMAT_%@" = "Today’s activities renew in %@";
+"AFTER_WEEK_1_EXPIRES_FORMAT_%@" = "Daily activities expire in %@";
+"AFTER_WEEK_1_RENEW_FORMAT_%@" = "Daily activities renew in %@";
 "WEEK_1_ACTIVITY_SLEEP" = "Sleep Check-In";
+"AFTER_WEEK_1_EXPIRES_WEEKLY_FORMAT_DAYS_%@" = "Weekly activities expire in %@ days";
+"AFTER_WEEK_1_EXPIRES_WEEKLY_FORMAT_HOURS_%@" = "Weekly activities expire in %@";
+"AFTER_WEEK_1_RENEW_WEEKLY_FORMAT_DAYS_%@" = "Weekly activities renew in %@ days";
+"AFTER_WEEK_1_RENEW_WEEKLY_FORMAT_HOURS_%@" = "Weekly activities renew in %@";
+"DONE_FOR_DAY" = "Done for day";
```
Comment (DanWebster) [inline BiomarinPKU/BiomarinPKU/PKU.strings]:
```suggestion
"DONE_FOR_DAY" = "Done for the day";
```

**Pull request merged**